Core/HLE/sceKernelModule.cpp

```diff
@@ -1720,7 +1720,7 @@ void __KernelLoadReset() {
 bool __KernelLoadExec(const char *filename, u32 paramPtr, std::string *error_string) {
 	SceKernelLoadExecParam param;
 
-	PSP_SetLoading("Loading exec...");
+	PSP_SetLoading("Loading executable...");
```

Comment:
I think we don't actually show these messages anymore.  Personally I think "exec" and "executable" wouldn't make much sense to users, but it doesn't matter since it's no longer shown.

-[Unknown]

Reply:
Oh, this reminded me something... And I found it: I prepared an issue a looong time ago but never posted it, I don't know why...
Here it is:

---

With [this pull request](https://github.com/hrydgard/ppsspp/pull/10715), new strings have been created for game launch information.
Now we have the following strings (if I haven't missed any):

```
Compiling shaders...
Generating code...
Loading executable...
Loading game...
Loading modules...
Loading shader cache...
Starting graphics...
Starting modules...
```

But [this pull request](https://github.com/hrydgard/ppsspp/pull/10986) replaced the display of these strings by a spinner.
We can also read a comment in this code by @unknownbrackets which says the following:

```
// Don't really need this, and it creates a lot of strings to translate...
// Maybe just show "Loading game..." only?
```

And he also [commented](https://github.com/hrydgard/ppsspp/issues/10974#issuecomment-385329450) this:

```
The text was intentionally removed because it causes a lot of extra strings to translate for 1.6.0.
```

But the strings mentioned above are still present in the code as of now.
So...
Is the spinner permanent and shown in all cases ? Therefore these strings would be useless ?
Or should we translate all these strings anyway because they will be reused ?

Reply:
The strings are still logged I think? But don't bother translating them, it's not useful. Won't show these to the user again for the beforementioned reasons.

Reply:
OK, thanks, I'll just delete them then.